atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/sweep/BackgroundSweeperImpl.java

```diff
@@ -42,6 +44,14 @@
 public final class BackgroundSweeperImpl implements BackgroundSweeper {
     private static final Logger log = LoggerFactory.getLogger(BackgroundSweeperImpl.class);
 
+    // We've noticed that Sweep increases GC pressure in cassandra when sweeping the VALUE table of a StreamStore.
+    // Thus, we use a reduced config in these scenarios.
+    private static final ImmutableSweepBatchConfig STREAM_STORE_BATCH_CONFIG = ImmutableSweepBatchConfig.builder()
+            .candidateBatchSize(128)
+            .deleteBatchSize(128)
+            .maxCellTsPairsToExamine(128)
+            .build();
```

Comment:
Hmm, I see that we do a `min` later on. The mechanics are a little funky though - if normal sweep goes 1024-512-256-128-64 etc it feels like this should go 128-64-32-16-8 but in actuality this looks like it goes 128-128-128-128-64.

Also I'm not sure how we want to behave if people config-override the (regular) sweep batch config. Feels like we should just have an additional penalty multiplier of 0.125 and apply that when sweeping stream-store value tables, making sure that we don't go below one cell.